.github/workflows/pulsar-ci.yaml

```diff
@@ -762,4 +858,4 @@ jobs:
 
       - name: Delete maven repository binaries from GitHub Actions Artifacts
         run: |
```

Comment:
`if: ${{ needs.changed_files_job.outputs.docs_only != 'true' }}` is needed here.